modules/apps/polls/polls-web/src/main/resources/META-INF/resources/polls/question_action.jsp

```diff
@@ -45,7 +47,7 @@ PollsQuestion question = (PollsQuestion)row.getObject();
 	<c:if test="<%= PollsQuestionPermission.contains(permissionChecker, question, ActionKeys.PERMISSIONS) %>">
 		<liferay-security:permissionsURL
 			modelResource="<%= PollsQuestion.class.getName() %>"
-			modelResourceDescription="<%= question.getTitle(locale) %>"
+			modelResourceDescription="<%= questionTitle.unsafeGetString() %>"
```

Comment:
hey @arthurchan35, 
I have concerns about this one. Since this is fed to `portletURL.setParameter` I expect it to properly escape it but, if that's not the case, I guess we should remove the unsafeGetString and add the proper context in the permissionURL tag